GettingStarted.md

```diff
@@ -126,17 +126,22 @@ In normal use, you shouldn't need to set this value, but this value can be used
 This project is using standard [Microsoft.Extensions.Logging](https://docs.microsoft.com/en-us/dotnet/core/extensions/logging?tabs=command-line).  To configure different log levels, please see the documentation at this link.
 
 ### Tracing and Monitoring Configuration
-Application monitoring and tracing can be configured using the `TELEMETRY_PROVIDER` variable in the application configuration
+Application monitoring and tracing can be configured using the `TelemetryProvider` variable in the application configuration.  
 
 ```bash
-TELEMETRY_PROVIDER= "ApplicationInsights"
+carbonAwareVars__TelemetryProvider= "ApplicationInsights"
```

Comment:
It should follow PascalCasing (i.e `CarbonAwareVars__TelemetryProvider`)
Also if this is bash, the `=` sign can't have spaces (i.e `CarbonAwareVars__TelemetryProvider="ApplicationInsights"`)

Reply:
Sounds good. I will also update the rest of the places where Pascal Casing is not followed